### PR TITLE
Fix sanitize_params method

### PR DESCRIPTION
**Pull request by mizinsky**
Permit only @smart_listing params.

### CODE REVIEW

**Comment by Ana06** [comment]
Is this fixing https://github.com/Sology/smart_listing/issues/134?

**Comment by mizinsky** [comment]
@Ana06 yes, this permit only smart-listing params.

**Comment by mizinsky** [comment]
I changed the approach, and there is no 'permit' method now.
I modeled on the kaminari's solution, and their way to handle strong params. 
[https://github.com/kaminari/kaminari/blob/e32c61cf6edd538bb9bca20421ec9447ebcd7d25/kaminari-core/lib/kaminari/helpers/tags.rb](url)

File: README.md [hunk 1]
```diff
@@ -238,8 +238,9 @@ When form field changes its value, form is submitted and request is made. This n
 ```ruby
 users_scope = User.active.joins(:stats)
 users_scope = users_scope.like(params[:filter]) if params[:filter]
-@users = smart_listing_create :users, users_scope, partial: "users/listing"
+@users = smart_listing_create :users, users_scope, partial: "users/listing", custom_params: [:filter]
```
Comment (ljachymczyk) [inline README.md]: @mizinsky is this still valid?

**Comment by ljachymczyk** [comment]
Thanks @mizinsky

**Pull request closed without merging**